beacon-chain/sync/validate_beacon_blocks.go

```diff
@@ -27,6 +27,10 @@ import (
 	"go.opencensus.io/trace"
 )
 
+var (
+	ErrOptimisticParent = errors.New("parent of the block is optimistic")
```

Comment:
Please add a godoc comment for all public things.

Reply:
aah.. it need not be public. But still i added comment to make it easy to understand.